### PR TITLE
Fix parsing of detailed timings

### PR DESCRIPTION
**Pull request by pp3345**
According to VESA E-EDID 1.4 Section 3.10.3, only the first three bytes
of a display descriptor need to be zeros. The fifth byte is usually
zero, however, may also contain a value in case the descriptor tag
indicates that this is a Display Range Limits Descriptor.

Furthermore, the third byte may very well be zero even if the descriptor
contains a detailed timing: this is in fact the case for screens with,
e.g., 2560 or 3840 horizontal addressable pixels (i.e., WQHD or 4K UHD),
where the lower 8 bits of the horizontal addressable pixels are zeroes
(see Section 3.10.2).

### CODE REVIEW

**Comment by mithro** [comment]
I can merge this, but it would be nice to have some sort of test. Do you have time to add one?

**Comment by pp3345** [comment]
Sure, will do that tomorrow.

**Comment by mithro** [comment]
@pp3345 - btw what is your interest in the project? Do you plan to do more work?

This project is really looking for someone to give it lots of love and would be more than happy to have you help drive that!

**Comment by pp3345** [comment]
Here is the promised test case.

Honestly, I am primarily interested in finding out how common it is for HDMI and DisplayPort monitors to have a _Video Capability Data Block_ in their CTA-861 extension block, so I also plan to work on CTA-861 parsing and maybe some other smaller stuff, though I can't really guarantee for anything right now.

File: edid_parser/tests.py [hunk 1]
```diff
@@ -286,6 +286,27 @@ def test_descriptors(self):
         self.assertIn('Additional_Color_Point', data)
         self.assertIn('Additional_Standard_Timing', data)
 
+        del self.parser.data['Monitor_Serial_Number']
+        del self.parser.data['Monitor_Name']
```
Comment (mithro) [inline edid_parser/tests.py]: Why are you deleting these things here?

Comment (pp3345) [reply edid_parser/tests.py]: Parsing the first and the second EDIDs would set these values. The EDID I'm adding to the test case also contains those. By deleting them before parsing again, we can ensure that the parser correctly recognizes those again in the third EDID.

Comment (pp3345) [reply edid_parser/tests.py]: Yeah, I totally agree.

**Pull request merged**